applications/GeoMechanicsApplication/custom_elements/Policies.md

```diff
@@ -0,0 +1,137 @@
+
+For a 3D element consisting of 4 nodes, the gradient looks like:
+```math
+\nabla{N} =
+\begin{bmatrix}
+\delta N_1/\delta x & \delta N_1/\delta y & \delta N_1/\delta z\\
+\delta N_2/\delta x & \delta N_2/\delta y & \delta N_2/\delta z\\
+\delta N_3/\delta x & \delta N_3/\delta y & \delta N_3/\delta z\\
+\delta N_4/\delta x & \delta N_4/\delta y & \delta N_4/\delta z
+\end{bmatrix}
+```
+Resulting in the following B-matrix:
+```math
+B =
+\begin{bmatrix}
+\delta N_1/\delta x & 0 & 0 & && & \delta N_4/\delta x & 0 & 0 \\
+0 & \delta N_1/\delta y & 0 & && & 0 & \delta N_4/\delta y & 0\\
+0 & 0 & \delta N_1/\delta z & &\dots& & 0 & 0 & \delta N_4/\delta z\\
+\delta N_1/\delta y & \delta N_1/\delta x & 0 & && & \delta N_4/\delta y & \delta N_4/\delta x & 0 \\
+0 & \delta N_1/\delta z & \delta N_1/\delta y & && & 0 & \delta N_4/\delta z & \delta N_4/\delta y \\
+\delta N_1/\delta z & 0 & \delta N_1/\delta x & && & \delta N_4/\delta z & 0 & \delta N_4/\delta x
+\end{bmatrix}
+```
+Where the first three columns are for the first node, the next three columns for the second node and so on for the number of nodes in the element.
+
+For a 2D element consisting of three nodes, the gradient looks like:
+```math
+\nabla{N} =
+\begin{bmatrix}
+\delta N_1/\delta x & \delta N_1/\delta y \\
+\delta N_2/\delta x & \delta N_2/\delta y \\
+\delta N_3/\delta x & \delta N_3/\delta y
+\end{bmatrix}
+```
+Where the rows depict the different nodes in the element (i.e. $N_i$ is the shape function for node $i$). For the plane strain case, this would result in the following B-matrix
+```math
+B =
+\begin{bmatrix}
+\delta N_1/\delta x & 0 & \delta N_2/\delta x & 0 & \delta N_3/\delta x & 0 \\
+0 & \delta N_1/\delta y & 0 & \delta N_2/\delta y & 0 & \delta N_3/\delta y\\
+0 & 0 & 0 & 0 & 0 & 0\\
+\delta N_1/\delta y & \delta N_1/\delta x & \delta N_2/\delta y & \delta N_2/\delta x & \delta N_3/\delta y & \delta N_3/\delta x
+\end{bmatrix}
+```
+For the axisymmetric stress state, this would result in the following B-matrix
+```math
+B =
+\begin{bmatrix}
+\delta N_1/\delta x & 0 & \delta N_2/\delta x & 0 & \delta N_3/\delta x & 0 \\
+0 & \delta N_1/\delta y & 0 & \delta N_2/\delta y & 0 & \delta N_3/\delta y\\
+N_1 / r & 0 & N_2 / r & 0 & N_3 / r & 0\\
+\delta N_1/\delta y & \delta N_1/\delta x & \delta N_2/\delta y & \delta N_2/\delta x & \delta N_3/\delta y & \delta N_3/\delta x
+\end{bmatrix}
+```
+
+### Integration coefficient
+
+For the 3D and plane strain stress states, the integration coefficient (or integration volume) is the product of the weight of the
+integration point and the determinant of the Jacobian:
+$$V_i = w_i \det{(J)}$$
+Where $w_i$ is the weight and $J$ is the Jacobian. For the axisymmetric stress state, this same calculation is done, but due to the symmetry, the integration volume
+is corrected by multiplying it with the axisymmetric circumference:
+$$V_i = 2\pi r w_i \det{(J)}$$
```

Comment:
and here w_i is an area. Why not to define w_i as a volume everywhere?

Reply:
If I understand it correctly now (please re-review this part, because I tried to explain it more clearly), it has to be an area at least for the axisymmetric case, since the volume depends on $r$.